### PR TITLE
[Mellanox] Add x86_64-nvidia_sn5610n-r0 new platform and SKUs

### PR DESCRIPTION
**Pull request by noaOrMlnx**
Add x86_64-nvidia_sn5610n-r0 Mellanox platform
##### Work item tracking
- Microsoft ADO **(number only)**:

### CODE REVIEW

**Comment by dprital** [comment]
@noaOrMlnx , Please add description to this PR

**Review by DavidZagury — APPROVED** [review]

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/sai_5610n_256x100g.xml
Comment (dprital) [inline device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/sai_5610n_256x100g.xml:1]: The file name does not reflect the hwSKU. 256x100 Vs 224o8

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/port_config.ini [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+Ethernet487    487                                    etp61h    61       100000   rs    off     8
+Ethernet496    496                                    etp63a    63       100000   rs    off     1
+Ethernet497    497                                    etp63b    63       100000   rs    off     2
+Ethernet498    498                                    etp63c    63       100000   rs    off     3
+Ethernet499    499                                    etp63d    63       100000   rs    off     4
+Ethernet500    500                                    etp63e    63       100000   rs    off     5
+Ethernet501    501                                    etp63f    63       100000   rs    off     6
+Ethernet502    502                                    etp63g    63       100000   rs    off     7
+Ethernet503    503                                    etp63h    63       100000   rs    off     8
+Ethernet512    512                                    etp65     65        10000   rs
```
Comment (dprital) [inline device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/port_config.ini:250]: Missing 2nd service port

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/media_settings.json
Comment (dprital) [inline device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/media_settings.json:1]: Should be symbolic link to the file we have on SN5600 (V256 hwSKU)

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/optics_si_settings.json
Comment (dprital) [inline device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/optics_si_settings.json:1]: Same as above. Symbolic link from SN5600

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/port_config.ini (hunk 1 above)
Comment (noaOrMlnx) [reply device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/port_config.ini:250]: done

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/sai_5610n_256x100g.xml
Comment (noaOrMlnx) [reply device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C224O8/sai_5610n_256x100g.xml:1]: done

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/optics_si_settings.json
Comment (noaOrMlnx) [reply device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/optics_si_settings.json:1]: done

File: device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/media_settings.json
Comment (noaOrMlnx) [reply device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/media_settings.json:1]: done

**Review by dprital — APPROVED** [review]

**Pull request closed without merging**